pkgs/applications/editors/android-studio/update.sh

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=./. -i bash -p python3Packages.yq nix-prefetch-scripts
+
+set -euo pipefail
+
+DEFAULT_NIX="$(realpath "./pkgs/applications/editors/android-studio/default.nix")"
+RELEASES_XML="$(curl -v --silent https://raw.githubusercontent.com/JetBrains/intellij-sdk-docs/main/topics/_generated/android_studio_releases.xml 2>/dev/null)"
+
+# Available channels: Release/Patch (stable), Beta, Canary
+getLatest() {
+    local attribute="$1"
+    local channel="$2"
+    [[ "$channel" = "stable" ]] && channel="release" # Our naming convention is different
+    local result="$(echo "$RELEASES_XML" \
+        | xq -r ".[].item[] | select(.channel == \"${channel^}\") | .${attribute}" \
+        | sort --version-sort \
+        | tail -n 1)"
+
+    if ! [[ -z "$result" ]]; then
```

Comment:
Just leaving this as a suggestion so I can merge it from githubs web UI.
```suggestion
    if [[ -n "$result" ]]; then
```